unittests/AST/ASTImporterTest.cpp

```diff
@@ -2466,6 +2466,47 @@ TEST_P(ImportFunctionTemplates,
   EXPECT_TRUE(ImportedD);
 }
 
+TEST_P(ImportFunctionTemplates, ImportFunctionTemplateInRecordDeclTwice) {
```

Comment:
Any of these failed before the change?

Reply:
The first fails, the second crashes with the `isFirst() && "decl became canonical unexpectedly"` assertion.